collector/cpu_freebsd.go

```diff
@@ -17,17 +17,14 @@ package collector
 
 import (
 	"errors"
-	"os"
+	"github.com/prometheus/client_golang/prometheus"
```

Comment:
nit: Would be nice to keep internal and external import separated as it was before